shopfloor_mobile_base/static/wms/src/i18n/en.json

```diff
@@ -0,0 +1,40 @@
+{
+  "cluster_picking.manual_selection.title": "Select a batch and start",
+  "cluster_picking.start_line.title": "Pick the product by scanning something",
+  "cluster_picking.start_line.scan_placeholder": "Scan location / pack / product / lot",
+  "cluster_picking.scan_destination.title": "Check qty and scan a destination bin",
+  "cluster_picking.scan_destination.scan_placeholder": "Scan destination bin",
+  "cluster_picking.unload_all.title": "Unload all bins",
+  "cluster_picking.confirm_unload_all.title": "Unload all bins confirm",
+  "cluster_picking.unload_single.title": "Unload single bin",
+  "cluster_picking.unload_set_destination.title": "Set destination",
+  "cluster_picking.confirm_unload_set_destination.title": "Set destination confirm",
+  "cluster_picking.change_pack_lot.title": "Change pack or lot",
+  "cluster_picking.change_pack_lot.scan_placeholder": "Scan pack or lot",
+  "location_content_transfer.scan_location.title": "Start by scanning a location",
+  "location_content_transfer.scan_destination_all.title": "Scan destination location for all items",
+  "location_content_transfer.start_single.scan_placeholder": "Scan pack / product / lot",
+  "location_content_transfer.scan_destination.title": "Set a qty and scan destination location",
+  "scan_placeholder_translation": "Scan location",
```

Comment:
All these entries comes from shopfloor_mobile and should not be declared into shopfloor_mobile_base